src/lib/parser.spec.ts

```diff
@@ -223,6 +234,7 @@ enum csvFixtures {
   inflowOutflowColumns = 9,
   githubIssue45 = 10,
   inflow0 = 11,
+  inflowOutflow0 = 12
```

Comment:
PS: If you run `yarn lint:fix`, it will fix this linting error. Then the tests will pass and this PR can be merged.